js/libs/keycloak-admin-client/src/resources/clients.ts

```diff
@@ -113,7 +113,7 @@ export class Clients extends Resource<{ realm?: string }> {
 
   public findRole = this.makeRequest<
     { id: string; roleName: string },
-    RoleRepresentation
+    RoleRepresentation | null
```

Comment:
You can trace it back to https://github.com/keycloak/keycloak/blob/main/js/libs/keycloak-admin-client/src/utils/fetchWithError.ts#L39
Since there is no code involved in the conversion it can only be null. 
There's an off-chance that it returns an empty string if the response body wasn't present. If it's a method that expects a response this should rather throw, but that's another issue

Reply:
I see, so sometimes the APi will just return the literal value `null`? That's strange, but not surprising.